README.md

```diff
@@ -95,6 +95,11 @@ If you checked out the project from GitHub you can build the project with maven
 ```
 mvn clean install
 ```
+If you want to generate the jar from GitHub you run the following maven command:
+```
+mvn package
```

Comment:
Given that you're editing this file, could you update this (and other places) to use `mvnw` instead of `mvn` please?